src/main/java/org/apache/commons/compress/archivers/zip/ZipMethod.java

```diff
@@ -124,6 +124,24 @@ public enum ZipMethod {
      */
     LZMA(14),
 
+    /**
+     * Compression Method 20 for Zstd.
+     *
+     * @see <a href="https://github.com/facebook/zstd">https://github.com/facebook/zstd</a>
+     * @see <a href="https://pkwaredownloads.blob.core.windows.net/pkware-general/Documentation/APPNOTE-6.3.7.TXT">deprecated zstd compression method id</a>
+     * @see <a href="https://www.pkware.com/documents/casestudies/APPNOTE.TXT">Explanation of fields: compression method: (2 bytes)</a>
+     */
```

Comment:
All new public and protected elements should carry a Javadoc `@since` tag, in this case 1.28.0. Done in git master.